recipes/docutils/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{%set name = "docutils" %}
+{%set version = "0.12" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "c7db717810ab6965f66c8cf0398a98c9d8df982da39b4cd7f162911eb89596fa" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --record=record.txt
```

Comment:
No? I added it because it was part of the standard `python setup.py install --single-version-externally-managed --record=record.txt`; it's a cargo-cult build setup for me. I can drop it, no problem.